mod-template-engine/src/main/resources/domain/mod-template-engine/features/templates.feature

```diff
@@ -12,42 +14,106 @@ Feature: Templates tests
 
   # CRUD
 
-  @Undefined
   Scenario: Post should return 201 and new template
-    * print 'undefined'
+    * def requestEntity = read('samples/template-entity.json')
+
+    Given path 'templates'
+    And request requestEntity
+    When method POST
+    Then status 201
+
+    Given path 'templates/' + templateId
+    When method GET
+    Then status 200
+    And match $.description == requestEntity.description
 
-  @Undefined
   Scenario: Post should return 400 if template resolver is not supported
-    * print 'undefined'
+    * def requestEntity = read('samples/template-entity.json')
+    * requestEntity.templateResolver = {}
+
+    Given path 'templates'
+    And request requestEntity
+    When method POST
+    Then status 400
 
-  @Undefined
   Scenario: Post should return 422 if template did not pass validation
-    * print 'undefined'
+    * def requestEntity = read('samples/invalid-template-entity.json')
```

Comment:
invalid-template-request-entity? please modify the entity, now it looks like valid